static/src/inline-svgs/journalism/in-article-audio-player/btn-pause.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Mind quickly dropping these SVGs in [there](https://jakearchibald.github.io/svgomg/)?